Add RHEL commands and clarify full-history re-sync time (#3719)
XRPLF/xrpl-dev-portal: docs/infrastructure/installation/migrate-to-xrpld.md

### 2. Stop and remove rippled

Once your backups are safely off the host, stop the service:

```sh
sudo systemctl stop rippled
```

Then remove the package. For Debian-based distros:

```sh
sudo apt-get remove -y rippled
```

For RHEL-based distros:

```sh
sudo yum remove -y rippled
```

{% admonition type="info" name="remove keeps your config" %}

> `apt-get remove` (and `yum remove`) leaves your config and data in place. `apt-get purge` would delete the config, which is why you back up first.

Reviewer: I think it is not very safe to say, yum remove does not remove config and data. It's better to just back up config and data.

#### Keep existing data on full-history nodes

> For full-history and large-history nodes, where re-downloading the ledger store from peers can take many months to years - a full-history backfill is impractical to rebuild from scratch. The simplest and safest option is to leave your data where it is and hand ownership to the new `xrpld` user. The config you restored in Step 4 already points at `/var/lib/rippled` and `/var/log/rippled`, so no paths need to change:

Reviewer 2: ```suggestion
Backfilling large amounts of ledger history is impractical; re-downloading the ledger store from peers can take months to years for full history. The simplest and safest option is to leave your data where it is and hand ownership to the new `xrpld` user. The config you restored in Step 4 already points at `/var/lib/rippled` and `/var/log/rippled`, so no paths need to change:
```
